src/lib.rs

```diff
@@ -105,11 +198,21 @@ fn value_to_pyobject(value: Value) -> PyResult<PyObject> {
 fn pyobject_to_value<'v>(obj: PyObject, heap: &'v Heap) -> PyResult<Value<'v>> {
```

Comment:
I know this hasn't been touched, but the lifetimes can be elided here:
```suggestion
fn pyobject_to_value(obj: PyObject, heap: &Heap) -> PyResult<Value> {
```